libopenage/CMakeLists.txt

```diff
@@ -164,4 +164,4 @@ if(NOT APPLE)
 	)
 endif()
 
-install(TARGETS libopenage LIBRARY DESTINATION lib)
+install(TARGETS libopenage DESTINATION lib LIBRARY DESTINATION lib)
```

Comment:
according to https://cmake.org/cmake/help/v3.4/command/install.html#installing-targets it shouldn't even work specifying destination before the type.